nix/templates/default/default.nix

```diff
@@ -42,38 +45,48 @@ naersk.lib."${targetPlatform.system}".buildPackage rec {
   LIBCLANG_PATH = "${llvmPackages.libclang.lib}/lib";
   buildInputs = [
     rustfmt
-    cargo-pgx
-    pkg-config
     cargo
     rustc
+    cargo-pgx
+    pkg-config
     libiconv
-    postgresql_13
+    pgxPostgresPkg
+    gcc
   ];
   checkInputs = [ cargo-pgx cargo rustc ];
   doCheck = true;
 
+  postPatch = "patchShebangs .";
   preConfigure = ''
```

Comment:
Could it init only for the selected versions? It can wait.

Reply:
Ah, I was intending to try that here! I guess a future PR. 🤦‍♀️